diff --git a/packages/binutils-libs/build.sh b/packages/binutils-libs/build.sh
index 5486721b45..d6aa25d5b1 100644
--- a/packages/binutils-libs/build.sh
+++ b/packages/binutils-libs/build.sh
@@ -6,7 +6,7 @@ TERMUX_PKG_VERSION="2.44"
 TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL=https://ftp.gnu.org/gnu/binutils/binutils-with-gold-${TERMUX_PKG_VERSION}.tar.xz
 TERMUX_PKG_SHA256=67be9198476cc37436e2801de649f4ad80bf0d02430d86aff63c6b59b6e23987
-TERMUX_PKG_DEPENDS="zlib, zstd"
+TERMUX_PKG_DEPENDS="zlib, zstd, libandroid-spawn, libandroid-glob"
 TERMUX_PKG_BREAKS="binutils (<< 2.39), binutils-dev"
 TERMUX_PKG_REPLACES="binutils (<< 2.39), binutils-dev"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
