diff --git a/packages/libuv/build.sh b/packages/libuv/build.sh
index 6d23c41527..bd9df08e93 100644
--- a/packages/libuv/build.sh
+++ b/packages/libuv/build.sh
@@ -7,6 +7,7 @@ TERMUX_PKG_VERSION="1.51.0"
 TERMUX_PKG_SRCURL=https://dist.libuv.org/dist/v${TERMUX_PKG_VERSION}/libuv-v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=5f0557b90b1106de71951a3c3931de5e0430d78da1d9a10287ebc7a3f78ef8eb
 TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_FORCE_CMAKE=true
 TERMUX_PKG_BREAKS="libuv-dev"
 TERMUX_PKG_REPLACES="libuv-dev"
 
