diff --git a/packages/openjdk-21/build.sh b/packages/openjdk-21/build.sh
index 0390aeaf4e..bd0586c682 100644
--- a/packages/openjdk-21/build.sh
+++ b/packages/openjdk-21/build.sh
@@ -95,6 +95,23 @@ termux_step_make_install() {
 	mkdir -p $TERMUX_PREFIX/lib/jvm/java-21-openjdk/etc/profile.d
 	echo "export JAVA_HOME=$TERMUX_PREFIX/lib/jvm/java-21-openjdk/" > \
 		$TERMUX_PREFIX/lib/jvm/java-21-openjdk/etc/profile.d/java.sh
+	ln -sf "$TERMUX_PREFIX/lib/jvm/java-21-openjdk/etc/profile.d/java.sh" "$TERMUX_PREFIX/etc/profile.d/java.sh"
+
+	# Remove unnecessary files
+	rm -vrf "$TERMUX_PREFIX/lib/jvm/java-21-openjdk/demo"
+
+	# Strip binaries
+	"$STRIP" $TERMUX_PREFIX/lib/jvm/java-21-openjdk/bin/*
+
+	# Symlink binaries
+	for bin in $TERMUX_PREFIX/lib/jvm/java-21-openjdk/bin/*; do
+		ln -sf "$bin" "$TERMUX_PREFIX/bin/$(basename "$bin")"
+	done
+
+	# Symlink manpages
+	for page in $TERMUX_PREFIX/lib/jvm/java-21-openjdk/man/man1/*; do
+		ln -sf "$page" "$TERMUX_PREFIX/share/man/man1/$(basename "$page")"
+	done
 }
 
 termux_step_post_make_install() {
