diff --git a/packages/openjdk-17/build.sh b/packages/openjdk-17/build.sh
index 38695e5768..b01b37adb7 100644
--- a/packages/openjdk-17/build.sh
+++ b/packages/openjdk-17/build.sh
@@ -115,6 +115,12 @@ termux_step_make_install() {
 	mkdir -p $TERMUX_PREFIX/lib/jvm/java-17-openjdk/etc/profile.d
 	echo "export JAVA_HOME=$TERMUX_PREFIX/lib/jvm/java-17-openjdk/" > \
 		$TERMUX_PREFIX/lib/jvm/java-17-openjdk/etc/profile.d/java.sh
+
+	# Remove unnecessary files
+        rm -vrf "$TERMUX_PREFIX/lib/jvm/java-17-openjdk/demo"
+
+	# Strip binaries
+	"$STRIP" $TERMUX_PREFIX/lib/jvm/java-17-openjdk/bin/*
 }
 
 termux_step_post_make_install() {
